.travis.yml

```diff
@@ -26,98 +26,100 @@ branches:
   only:
     - master
 
+# TODO: Test on the minimum and maximum supported platform versions
+
 jobs:
   include:
     - stage: Node tests
-      osx_image: xcode10.2
+      osx_image: xcode10
```

Comment:
This isn't needed if it matches the overall version (from line 4).

Reply:
OK, I'll remove that for tests, which are supposed to run on the minimum supported version

Reply:
Makes it easier for the future.